charts/jenkins/README.md

```diff
@@ -1012,6 +1012,36 @@ Upgrade an existing release from `stable/jenkins` to `jenkins/jenkins` seamlessl
 
 Chart release versions follow [SemVer](../../CONTRIBUTING.md#versioning), where a MAJOR version change (example `1.0.0` -> `2.0.0`) indicates an incompatible breaking change needing manual actions.
 
+### To 5.0.0
```

Comment:
the README is so long anyway I would put a note here and create a separate upgrade guide I think